webpack.config.ts

```diff
@@ -383,6 +384,20 @@ const appConfig: Configuration = {
       /moment\/locale/,
       new RegExp(`(${supportedLanguages.join('|')})\\.js$`)
     ),
+
+    /**
+     * Copies file logo-sentry.svg to the dist/entrypoints directory so that it can be accessed by
+     * the backend
+     */
+    new CopyPlugin({
+      patterns: [
+        {
+          from: `${staticPrefix}/images/logo-sentry.svg`,
+          to: 'entrypoints/logo-sentry.svg',
```

Comment:
Add copy-webpack-plugin to webpack config so that logo-sentry.svg can be copied from `static/images` to `entrypoints` on webpack build.
It is copied to the `entrypoints` instead of `assets` because assets with a contenthash cannot be accessed by the backend.